tasks/kmt.py

```diff
@@ -617,6 +687,139 @@ def build_run_config(run: str | None, packages: list[str]):
     return c
 
 
+def build_target_packages(filter_packages):
+    all_packages = go_package_dirs(TEST_PACKAGES_LIST, [NPM_TAG, BPF_TAG])
+    if filter_packages == []:
+        return all_packages
+
+    filtered = list()
+    for pkg in all_packages:
+        if os.path.relpath(pkg) in filter_packages:
+            filtered.append(pkg)
+
+    return filtered
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    return [pkg for pkg in all_packages if os.path.relpath(pkg) in filter_packages]
```